### PR TITLE
[17.0][PW] auth_api_key: Port from 14.0

### PR DESCRIPTION
**Pull request by thienvh332**
### Ported PRs and commits:
- `github.com/oca/commit/76955cff`
- `https://github.com/OCA/server-auth/pull/501`

### CODE REVIEW

File: auth_api_key/migrations/14.0.3.0.0/pre-migrate.py
Comment (nilshamerlinck) [inline auth_api_key/migrations/14.0.3.0.0/pre-migrate.py:1]: should be removed no?

Comment (thienvh332) [reply auth_api_key/migrations/14.0.3.0.0/pre-migrate.py:1]: I removed it in commit migration to 18.0

Comment (thienvh332) [reply auth_api_key/migrations/14.0.3.0.0/pre-migrate.py:1]: This PR is just to show the commits we will pick. And the commit processing will be done in commit migration.

Comment (nilshamerlinck) [reply auth_api_key/migrations/14.0.3.0.0/pre-migrate.py:1]: oh, why not 2 PRs
- 1 for branch 17.0
- 1 for branch 18.0

Comment (thienvh332) [reply auth_api_key/migrations/14.0.3.0.0/pre-migrate.py:1]: You mean we will create both PRs on OCA right? I misunderstood it as PR on 17.0 just for easy review. I will fix it

Comment (nilshamerlinck) [reply auth_api_key/migrations/14.0.3.0.0/pre-migrate.py:1]: yes, I think it's worth having both

Comment (thienvh332) [reply auth_api_key/migrations/14.0.3.0.0/pre-migrate.py:1]: I updated it. Thanks nils

**Pull request closed without merging**